packages/jest-cli/src/TestRunner.js

```diff
@@ -212,8 +215,9 @@ class TestRunner {
 
     const testRun =
       shouldRunInBand()
-      ? this._createInBandTestRun(testPaths, onTestResult, onRunFailure)
-      : this._createParallelTestRun(testPaths, onTestResult, onRunFailure);
+      ? this._createInBandTestRun(testPaths, argv, onTestResult, onRunFailure)
+      :
+      this._createParallelTestRun(testPaths, argv, onTestResult, onRunFailure);
```

Comment:
nit: this style isn't great. Let's rename `onTestResult` to `onResult` and `onRunFailure` to `onFailure`. That should be sufficiently descriptive and allow you to bring this back into 80 chars.